src/js/src/components/SearchSingleItemComponents/SearchSingleItemAllData.vue

```diff
@@ -5,51 +5,121 @@
         {{ allDataButtonText }}
       </button>
     </div>
-    <div v-if="allDataShown">
+    <div v-if="allDataShown && allData !== {}">
       <hr class="informationDivider">
       <div class="table">
         <div class="tr">
           <span class="td highlightText">Attribute</span><span class="td highlightText">Value</span>
         </div>
-        <div v-for="(key, index) in Object.entries(result)" :key="index" class="tr">
-          <span class="td">{{ key[0] }}</span> <span class="td">{{ key[1] }}</span>
+        <div v-for="(key, index) in Object.entries(allData)"
+             :key="index"
+             class="tr">
+          <span class="td">{{ key[0] }}</span>
+          <div v-if="key[1].constructor === Array"
+               :class="key[0] !== 'content' ? 'td content clickAble' : 'td content'">
+            <span v-for="(entry, newIndex) in key[1]"
+                  :key="newIndex"
+                  :class="index === currentDataShown ? 'singleEntry' : newIndex < arrayShownLimit ? 'singleEntry' : 'singleEntry hidden'"
+                  @click="key[0] !== 'content' ? searchFromAllValues(key[0], entry) : null">
+              {{ entry }} <br>
+            </span>
+            <button v-if="key[1].length > arrayShownLimit"
+                    :key="index + '-button' "
+                    class="attributeButton"
+                    @click="toggleShownData(index)">
+              {{ specificValueButtonText(index) }}
+            </button>
+          </div>
+          <div v-if="key[1].constructor !== Array" :class="key[0] !== 'content' ? 'td content clickAble' : 'td content'" @click="key[0] !== 'content' ? searchFromAllValues(key[0], key[1]) : null">
+            <span :class="key[0] === 'content' ? '' : 'singleEntry'"> {{ key[0] === 'content' ? displayContentValue(key[1]) : key[1] }}</span>
+          </div>
```

Comment:
So much ternary action in this template...any way to at least simplify the ternary in ternary stuff?

Reply:
I tried to make it more readable now.

Reply:
👍